client/src/components/DailyCardFasting.js

```diff
@@ -0,0 +1,61 @@
+import styled from "styled-components/macro";
+
+const Container = styled.div`
+  width: 270px;
+  height: 65vh;
+  padding: 20px 40px;
+  border-radius: 30px;
+  background: #ffffff;
+  box-shadow: 30px 30px 60px #0d569f29, -10px -10px 70px #ffffff;
+  div::-webkit-scrollbar {
+    display: none;
+  }
+`;
+
+const Header = styled.div`
+  display: grid;
+  grid-template-columns: 1fr auto;
+  margin-bottom: 15px;
+`;
+
+const TextContainer = styled.div`
+  display: grid;
+  overflow: auto;
+  height: 52vh;
+  padding-right: 20px;
+`;
+
+const Headline = styled.p`
+  padding-bottom: 8px;
+`;
+
+const Content = styled.span`
+  padding-bottom: 20%;
+`;
```

Comment:
```suggestion
p {
  padding-bottom: 8px
  }

span {
  padding-bottom: 20%;
}
```
If your just using p and span like this inside your TextContainer component you can just make them a part of it instead of defining styled components for them :)

Reply:
Thanks for the suggestion!